### PR TITLE
Add draggable false to button markup and validation

### PR DESCRIPTION
**Pull request by owenatgov**
## What
Adds `draggable="false"` to the govspeak button extension output.

## Why
This is in response to a change to button markup in the components gem (https://github.com/alphagov/govuk_publishing_components/pull/2448) which wasn't reflected in govspeak. This caused the govspeak validator to assume all buttons failed as the in-app markup was including `draggable="false"` whilst sanitised govspeak wasn't.

[Zendesk ticket which prompted this](https://govuk.zendesk.com/agent/tickets/4779605)

No visual changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 6.7.6
```
Comment (karlbaker02) [inline CHANGELOG.md:1]: Is the convention that we make an `Unreleased` commit first and then make another commit to bump the version?

File: lib/govspeak.rb [hunk 2]
```diff
@@ -224,7 +224,7 @@ def insert_strong_inside_p(body, parser = Govspeak::Document)
       text = text.strip
       href = href.strip
 
-      %(\n<a role="button" class="#{button_classes}" href="#{href}" #{data_attribute}>#{text}</a>\n)
+      %(\n<a role="button" draggable="false" class="#{button_classes}" href="#{href}" #{data_attribute}>#{text}</a>\n)
```
Comment (karlbaker02) [inline lib/govspeak.rb:227]: We still need `draggable="false"` here right?

**Review by karlbaker02 — COMMENTED** [review]
Looks good, a few small comments and questions

File: CHANGELOG.md (hunk 1 above)
Comment (owenatgov) [reply CHANGELOG.md:1]: I was thinking this. I've been looking back at some old PRs and I reckon that convention makes sense. I'll update accordingly.

File: lib/govspeak.rb (hunk 2 above)
Comment (owenatgov) [reply lib/govspeak.rb:227]: After a very quick test, we indeed do.

**Review by karlbaker02 — APPROVED** [review]
Looks great, thanks Owen! 👍
